deploy/examples/cluster-on-pvc-minikube.yaml

```diff
@@ -90,6 +95,31 @@ spec:
               values:
                 - minikube
 ---
+kind: PersistentVolume
```

Comment:
as it is cluster-on-pvc yaml why we are creating pvs and pvc here?

Reply:
pvs are used to create OSDs and the pvc is used by the monitor